diff --git a/node_modules/jest-haste-map/build/crawlers/node.js b/node_modules/jest-haste-map/build/crawlers/node.js
index 5f82cb6..85dd7c0 100644
--- a/node_modules/jest-haste-map/build/crawlers/node.js
+++ b/node_modules/jest-haste-map/build/crawlers/node.js
@@ -168,9 +168,10 @@ function find(roots, extensions, ignore, enableSymlinks, callback) {
           }
 
           if (typeof entry !== 'string') {
-            if (entry.isSymbolicLink()) {
-              return;
-            }
+            // @xnerhu
+            // if (entry.isSymbolicLink()) {
+            //   return;
+            // }
 
             if (entry.isDirectory()) {
               search(file);
@@ -184,7 +185,7 @@ function find(roots, extensions, ignore, enableSymlinks, callback) {
             activeCalls--; // This logic is unnecessary for node > v10.10, but leaving it in
             // since we need it for backwards-compatibility still.
 
-            if (!err && stat && !stat.isSymbolicLink()) {
+            if (!err && stat) { // && !stat.isSymbolicLink() @xnerhu
               if (stat.isDirectory()) {
                 search(file);
               } else {
